src/main/java/ddingdong/ddingdongBE/file/dto/ExcelClubMemberDto.java

```diff
@@ -70,30 +64,31 @@ public static ClubMemberDto fromExcelRow(Row row) {
                 }
             } else if (cell.getCellType() == CellType.NUMERIC) {
                 if (cell.getNumericCellValue() != 0) {
```

Comment:
```suggestion
          if (cell.getCellType() == CellType.NUMERIC && cell.getNumericCellValue() != 0) {
```
위처럼 if문 하나에 조건을 걸어도 괜찮지 않을까요??
위 조건을 메소드로 따로 분리시켜도 가독성에 좋아보입니다!